auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.java

```diff
@@ -65,7 +65,7 @@ public void saveCredentials(@NonNull Credentials credentials) {
      */
     public void getCredentials(@NonNull final BaseCallback<Credentials, CredentialsManagerException> callback) {
         String accessToken = storage.retrieveString(KEY_ACCESS_TOKEN);
-        String refreshToken = storage.retrieveString(KEY_REFRESH_TOKEN);
+        final String refreshToken = storage.retrieveString(KEY_REFRESH_TOKEN);
```

Comment:
How come this one is now final?

Reply:
Because I use it on an anonymous function right below and needs any variable accessed from outside its scope to be final.  -> https://github.com/auth0/Auth0.Android/pull/118/files#diff-2651976c12b43b2192543b92b10ed2e4R91